### ISSUE TITLE
Ticket creation

### ISSUE THREAD

**Issue by joelhawksley** [mojotech/hive#30]
As a user, I should be able to create a Ticket for a Project. (Let's save the subclassing for another story)

A Ticket should have:

- [x] ID (auto_inc)
- [ ] Title
- [x] Description

UI:

- [ ] Ticket creation
- [ ] Ticket list view
- [ ] Ticket show view

**Comment by joelhawksley** [comment]
Blocked on https://github.com/mojotech/hive/issues/29
